app/styles/layouts/_l-default.scss

```diff
@@ -95,6 +95,11 @@ body {
   @include breakpoint(xlarge) {
     width: calc(100% - 18rem);
   }
+
+  &.full-width {
+      width: 100% !important;
```

Comment:
I really would prefer to not add any `!importants`. Future us will thank present us. Are you sure this is necessary? I took it out running your branch locally and things seem to still work.